org.hectabit.PageBurger.json

```diff
@@ -0,0 +1,30 @@
+{
+  "id": "org.hectabit.PageBurger",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "pageburger",
+  "finish-args": [
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
```

Comment:
Add `--device=dri`, shows a bunch of mesa related warnings.

Reply:
ok then